pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_helper.go

```diff
@@ -128,3 +133,66 @@ func isValidMigratedvSphereVolume(ctx context.Context, pvMetadata metav1.ObjectM
 	}
 	return false
 }
+
+// updateVolumeSnapshotAnnotations updates annotations passed as key-value pairs
+// on VolumeSnapshot object
+func (c *K8sOrchestrator) updateVolumeSnapshotAnnotations(ctx context.Context,
+	volumeSnapshotName string, volumeSnapshotNamespace string, volumeSnapshotAnnotations map[string]string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	retryCount := 0
+	interval := time.Second
+	limit := 5 * time.Minute
+	// TODO: make this configurable
```

Comment:
When are you going to make this configurable?  In a followup PR?

Reply:
+1. Better to make it configurable. Also define default values as constants if possible.

Reply:
This requires quite a few changes, I've added it to the internal task list and will address it later.